src/backend/sgx/thread.rs

```diff
@@ -161,6 +158,11 @@ impl super::super::Thread for Thread {
             _ => unreachable!(),
         }
 
+        if self.cssa > 4 {
```

Comment:
The commit message for this says:
```
 fix(backend/sgx): exit for CSSA > 3

Do not enter a CSSA > 3, but exit the whole process.
```
What explains the discrepancy between `> 4` in the code and `> 3` in the commit message?

Reply:
oops

Reply:
fixed